api/v1/paths/owners/{ownerId}.js

```diff
@@ -46,6 +46,10 @@ const deleteOwner = async (req, res) => {
     }
     return res.status(204).send(result);
   } catch (err) {
+    if (err.errorNum === 2292) {
+      return errorBuilder(res, 409, 'This owner has one or more parks associated with it. '
+      + 'Please change the owner of each of the parks before deleting the owner.');
```

Comment:
Could you join these lines into one line? https://github.com/airbnb/javascript#strings--line-length